.travis.yml

```diff
@@ -0,0 +1,51 @@
+sudo: required
+dist: xenial
+language: go
+
+go: "1.14.x"
+
+services:
+  - docker
+
+branches:
+  only:
+    - master
+
+env:
+  - GO111MODULE=on
```

Comment:
This shouldn't be necessary anymore: since 1.13, Go should use Go modules if a `go.mod` file exists.